core/src/main/java/io/questdb/std/Unsafe.java

```diff
@@ -465,6 +470,7 @@ public Class<?> define(Class<?> hostClass, byte[] data) {
                 throw new InstantiationException("failed to initialize class definer");
             }
             anonymousClassDefiner = classDefiner;
+            enableJmalloc = !Os.isWindows() && System.getenv("no_jemalloc") == null;
```

Comment:
nit: `QDB_NO_JEMALLOC`?

Reply:
done in https://github.com/questdb/questdb/pull/4451/commits/9eca97fe4a54d2882613a9804ce95f3a37ffa9c2